src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/custom.py

```diff
@@ -101,7 +101,7 @@ def set_storage_account_properties(
 def upload_blob( # pylint: disable=too-many-locals
         client, container_name, blob_name, file_path, blob_type=None,
         content_settings=None, metadata=None, validate_content=False, maxsize_condition=None,
-        max_connections=2, max_retries=5, retry_wait=1, lease_id=None, if_modified_since=None,
```

Comment:
>max_retries=5, retry_wait=1 [](start = 27, length = 27)

are these gone?  It seems strange that there is a breaking change here with features removed